client/components/web-preview/content.jsx

```diff
@@ -65,6 +72,17 @@ export default class WebPreviewContent extends Component {
 		if ( this.props.showPreview && ! prevProps.showPreview && this.state.loaded ) {
 			this.focusIfNeeded();
 		}
+
+		// If the fixedViewportWidth changes, re-calculate the iframe styles.
+		if ( this.props.fixedViewportWidth !== prevProps.fixedViewportWidth ) {
+			if ( this.props.fixedViewportWidth ) {
+				this.handleResize();
+				window.addEventListener( 'resize', this.handleResize );
```

Comment:
I think we have to check whether the previous one is `undefined` or not. If not, I think we don't need to register this event again. Otherwise, we'll register this event multiple times. In short, we have to register this event only when the previous one is `undefined`.